### ISSUE TITLE
Validate security group descriptions on format and length

### ISSUE THREAD

**Issue by kgeisink** [awslabs/cfn-python-lint#173]
*cfn-lint version: 0.3.1*

It's a tiny one, but security group descriptions have a specific notation that do not allow all characters. `cloudformation validate` also does not catch these, causing the CloudFormation deployment to error.

The following error is provided, which shows the accepted format and property conditions:
```
Invalid rule description. Valid descriptions are strings less than 256 characters from the following set: a-zA-Z0-9. _-:/()#,@[]+=&;{}!$* (Service: AmazonEC2; Status Code: 400; Error Code: InvalidParameterValue; Request ID: a834965c-25c0-4ec7-86a3-099f6970bc0a)
```

A Description such as `This is AWS' own validator's fault` would trigger it as the `'` character is not supported.

**Comment by cmmeyer** [comment]
Resolved with #177, closing issue.  Thanks for the suggestion!
